test/lib/accumulate_distribute/meta/init_state.js

```diff
@@ -18,6 +18,17 @@ describe('accumulate_distribute:meta:init_state', () => {
     stubOrderAmountsGen.restore()
   })
 
+  it('initializes order amounts, floats', () => {
+    const stubOrderAmountsGen = sinon.stub(orderAmounts, 'gen').returns([0.001, 0.002])
```

Comment:
if `gen.orderAmounts` is stubbed, what float function are we testing here then?

might be missing something, but i think we can delete this test